Classes/MOC/Varnish/Package.php

```diff
@@ -2,18 +2,26 @@
 namespace MOC\Varnish;
 
 use TYPO3\Flow\Package\Package as BasePackage;
+use TYPO3\Flow\Annotations as Flow;
 
 class Package extends BasePackage {
 
 	/**
+     * @Flow\InjectConfiguration("enabled")
+     * @var boolean
+     */
+    protected $enabled;
```

Comment:
As far as i cann see it is not possible to Inject Configuration directly into the Package.php should i revert the changes to MOC\Varnish\Service\ContentCacheFlusherService->flushForNode ?